en/core-libraries/components/pagination.rst

```diff
@@ -90,6 +90,11 @@ array after the model you wish to configure::
 The values of the ``Post`` and ``Author`` keys could contain all the properties
 that a model/key less ``$paginate`` array could.
 
+.. note::
+    The ``$paginate`` variable is not automatically passed to the :php:class:`PaginatorComponent`'s settings. You must explicitly set the :php:class:`PaginatorComponent`'s settings using the following line::
+
+        $this->Paginator->settings = $this->paginate;
```

Comment:
Ignoring the helper side of things. This was more to do with the component and the way it's documented. I could be wrong, but doesn't $this->paginate have to be passed to the paginatorComponent before you can use $this->paginate() with those settings?

Naming the property $this->paginate, and then calling $this->Paginator->paginate, may imply that there is some magic from Cake. Where as you actually need to call $this->Paginator->settings = $this->paginate; first, for those settings to be taken into account. 

It was more for clarification, in case somebody was using $this->paginate and wondering why it was not working.